src/linker/configuration.py

```diff
@@ -210,22 +168,50 @@ def _get_required_attribute(environment: Dict[Any, Any], key: str) -> str:
         return value
 
     @staticmethod
-    def _get_requests(environment: Dict[Any, Any], key: str) -> Dict[Any, Any]:
+    def _get_requests(
```

Comment:
This is extremy hacky/ugly/terrible. But we have a ticket to just use vivarium's ConfigTree so I figured this is fine for now